corehq/apps/domain/management/commands/delete_domain.py

```diff
@@ -74,6 +75,11 @@ def hard_delete_forms(domain_name):
         for chunk in chunked(form_ids, 1000, list):
             XFormInstance.objects.hard_delete_forms(domain_name, chunk)
 
+    @staticmethod
+    def hard_delete_groups(domain_name):
```

Comment:
where is this called?

Reply:
Thanks it should be called in delete_domain right next to
```
self.hard_delete_cases(domain_name)
self.hard_delete_forms(domain_name)
```

Cal also brought up an interesting point that it seems like at some point we would just want to iterate through ES indices, and ensure anything related to the current domain is deleted so I'm still thinking about that.